setup.py

```diff
@@ -28,6 +25,14 @@
     os.system("python manage.py test")
     sys.exit()
 
+if sys.argv[-1] == "reformat":
+    os.system("isort -rc bootstrap3")
+    os.system("isort -rc demo")
+    os.system("autoflake -ir bootstrap3 demo --remove-all-unused-imports")
+    os.system("black .")
+    os.system("flake8 bootstrap3 demo")
+    sys.exit()
+
```

Comment:
this also ends in user installations of the package (not really a problem, but slightly messy). You might want to consider moving the 'build'-related code to separate script

for example: https://github.com/jieter/django-tables2/blob/master/maintenance.py

Reply:
In that case why not a Makefile or fabfile? A python script that does `os.sys` calls seems like the wrong tool for the job (but it gets it done).

Reply:
I turned it into a Makefile